appveyor.yml

```diff
@@ -83,6 +83,7 @@ test_script:
     rmdir /s /q cbmc\byte_update7
     rmdir /s /q cbmc\pipe1
     rmdir /s /q cbmc\unsigned___int128
+    rmdir /s /q cbmc-cpp
```

Comment:
Would this work if we had #1260 merged?

Reply:
Yes. You should be able to merge this PR and #1260 in any order. I'd recommend merging them close to each other.

Reply:
How about merging this one first, and then reverting this particular change in #1260?

Reply:
Sorry, I misunderstood. The tests in cbmc-cpp tests never ran on any platform. This PR enables in general, and then disables them specifically for windows. They don't work on windows irrespective of #1260, since they #include <assert.h>.